pkg/observ/observ.go

```diff
@@ -43,13 +84,30 @@ func RegisterTraceExporter(URL, service, ENV string) (*(jaeger.Exporter), error)
 		return nil, errors.E(op, err)
 	}
 
-	// And now finally register it as a Trace Exporter
-	trace.RegisterExporter(je)
+	trace.RegisterExporter(ex)
```

Comment:
like i said in my previous comments, let's put this line and the following 3 lines in a function so that we don't repeat it in every implementation.